armadillo/src/main/java/at/favre/lib/armadillo/OnSecurePreferenceChangeListener.java

```diff
@@ -0,0 +1,34 @@
+package at.favre.lib.armadillo;
+
+import android.content.SharedPreferences;
+import android.support.annotation.NonNull;
+
+/**
+ * Allow to register preference change listeners that receive
+ * @since 03.07.2019
+ */
+public interface OnSecurePreferenceChangeListener {
+
+    /**
+     * Variation of the regular {@link android.content.SharedPreferences.OnSharedPreferenceChangeListener#onSharedPreferenceChanged(SharedPreferences, String)} that
```

Comment:
Please add a simple code example in javadoc on how to use it.

`<pre> .... </pre>`